# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/target/lvp/patches/etc_profile.patch
# ROCK Linux is Copyright (C) 1998 - 2006 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

each directory on a iso9660 filesystem contains a file called TRANS.TBL
we don't want to source that one

--- profile	2003-10-21 16:26:28.000000000 +0200
+++ profile	2004-01-28 17:16:38.000000000 +0100
@@ -93,6 +93,7 @@
 # include local profiles
 #
 for x in /etc/profile.d/* /etc/conf/profile ; do
+	[ "${x##*/}" = "TRANS.TBL" ] && continue
 	[ -f ${x} ] && . ${x}
 done
 unset x
